app/src/common/shared/com/igalia/wolvic/ui/adapters/Bookmark.java

```diff
@@ -32,7 +32,7 @@ public Bookmark(@NonNull BookmarkNode node, int level, boolean isExpanded) {
         mTitle = node.getTitle();
         mURL = node.getUrl();
         mGuid = node.getGuid();
-        mPosition = node.getPosition() != null ? node.getPosition() : 0;
```

Comment:
I think I asked this before but how safe is this change?

Reply:
They are now using UInt for the position, so actually, I'm not sure if there's any proper way for tackling this except for rewriting this into Kotlin.

![image](https://github.com/Igalia/wolvic/assets/43995067/ecc334b2-c612-41be-b55b-de3a3a5af152)

![image](https://github.com/Igalia/wolvic/assets/43995067/5cc1cf56-8c5b-4140-9c20-0b3d74476395)

Reply:
Looks like currently the only place that would like to read the bookmark's position value is here:
 https://github.com/Igalia/wolvic/blob/e59cd9583bd744400193bcecd60fcf8aeaf5ee6f/app/src/common/shared/com/igalia/wolvic/ui/adapters/BookmarkAdapter.java#L324-L328

And that `getItemId()` has never been called from any `BookmarkAdapter` instances, so I believe currently we are perfectly safe to leave it just as it is.

I will add comments reminding people that the `getItemId()` and `getPosition()` methods are actually broken.